db/migrate/20161211075744_add_source_and_destination_to_messages.rb

```diff
@@ -1,6 +1,6 @@
 class AddSourceAndDestinationToMessages < ActiveRecord::Migration[5.0]
   def change
-    add_column :messages, :source, :string, null: false
-    add_column :messages, :destination, :string, null: false
+    #add_column :messages, :source, :string, null: false
+    #add_column :messages, :destination, :string, null: false
```

Comment:
this was moved to the table creation - sqlite is unable to 'add_column' with NOT_NULL and no default value - i'll probably just remove the migration